pkg/cmd/cli/cmd/rsync/rsync.go

```diff
@@ -111,6 +117,7 @@ func NewCmdRsync(name, parent string, f *clientcmd.Factory, out, errOut io.Write
 	cmd.Flags().StringVar(&o.RsyncInclude, "include", "", "rsync - include files matching specified pattern")
 	cmd.Flags().BoolVar(&o.RsyncProgress, "progress", false, "rsync - show progress during transfer")
 	cmd.Flags().BoolVar(&o.RsyncNoPerms, "no-perms", false, "rsync - do not transfer permissions")
+	cmd.Flags().BoolVarP(&o.Watch, "watch", "w", false, "Watch directory for changes and resync automatically")
```

Comment:
It's fine, just wanted to check

On Tue, Apr 26, 2016 at 12:42 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/rsync/rsync.go
> <https://github.com/openshift/origin/pull/8268#discussion_r61120407>:
>
> > @@ -111,6 +117,7 @@ func NewCmdRsync(name, parent string, f *clientcmd.Factory, out, errOut io.Write
> >  	cmd.Flags().StringVar(&o.RsyncInclude, "include", "", "rsync - include files matching specified pattern")
> >  	cmd.Flags().BoolVar(&o.RsyncProgress, "progress", false, "rsync - show progress during transfer")
> >  	cmd.Flags().BoolVar(&o.RsyncNoPerms, "no-perms", false, "rsync - do not transfer permissions")
> > +	cmd.Flags().BoolVarP(&o.Watch, "watch", "w", false, "Watch directory for changes and resync automatically")
>
> would we ever want the equivalent of get --watch functionality in rsync? I
> have a hard time envisioning that we would.
>
> by which i mean, i have a hard time envisioning what that would mean/do.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8268/files/b9f4b16543df4fb40bdd86f2797f6d2dd2e14093..5b20fcd4b5c6e42bdf67cf7b118d57d95e532e6f#r61120407>
>